x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/__jest__/processors/bytes.test.tsx

```diff
@@ -85,7 +79,47 @@ describe('Processor: Bytes', () => {
     const processors = getProcessorValue(onUpdate, BYTES_TYPE);
     expect(processors[0].bytes).toEqual({
       field: 'field_1',
-      ...defaultBytesParameters,
+    });
+  });
+
+  test('saves with common fields set', async () => {
+    const {
+      actions: { addProcessor, saveNewProcessor, addProcessorType },
+      form,
+      find,
+    } = testBed;
+
+    // This test ensures that the common fields that are used across all processors
+    // works and removes the need for those fields to be in every processors' test.
+
+    // Open flyout to add new processor
+    addProcessor();
+    // Add type (the other fields are not visible until a type is selected)
+    await addProcessorType(BYTES_TYPE);
+    // Add "field" value (required)
+    form.setInputValue('fieldNameField.input', 'field_1');
+
+    form.toggleEuiSwitch('ignoreFailureSwitch.input');
+
+    form.setInputValue('targetField.input', 'target_field');
+
+    form.setInputValue('tagField.input', 'some_tag');
+
+    // Trying to get the monaco text editor to work with CITs.
```

Comment:
Is this comment still needed? It appears it is working as expected now.

Reply:
Thanks. Removed.